### PR TITLE
specify bash shell for windows CI steps needing envs

### PR DESCRIPTION
**Pull request by bgentry**

### CODE REVIEW

**Comment by bgentry** [comment]
@brandur this should be just about there, just dealing with the fact that graceful shutdowns with sigterm aren't a thing on windows and we'd need to come up with another way to do this cleanly.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -242,56 +259,3 @@ jobs:
         run: |
           echo "Make sure that all sqlc changes are checked in"
           make verify/sqlc
-
-  producer_sample:
```
Comment (brandur) [inline .github/workflows/ci.yaml:246]: Just to make sure: you took this out on purpose?

**Review by brandur — APPROVED** [review]
Great!

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (bgentry) [reply .github/workflows/ci.yaml:246]: Yeah, I don’t think it’s needed anymore. We have enough coverage elsewhere imo.

**Pull request merged**